packages/language-server/src/plugins/svelte/features/getDiagnostics.ts

```diff
@@ -91,6 +91,13 @@ async function createParserErrorDiagnostic(error: any, document: Document) {
             document.scriptInfo || document.moduleScriptInfo
         );
 
+        if (!document.config?.preprocess && document.getText().includes('lang="')) {
```

Comment:
Would it be better if we use the `TagInformation.attribute.lang` here? if the `defaultLanguages` is set. It should already have the preprocess config so the attribute should accurate?

Also, should we silence the error when the fallback config is used? If so it might need more check as the preprocess config won't be falsy here.

Reply:
You are right! Will enhance this.